ratis-server/src/main/java/org/apache/ratis/server/storage/RaftStorageImpl.java

```diff
@@ -143,8 +143,10 @@ public RaftConfiguration readRaftConfiguration() {
     try (FileInputStream fio = new FileInputStream(confFile)) {
       LogEntryProto confProto = LogEntryProto.newBuilder().mergeFrom(fio).build();
       return LogProtoUtils.toRaftConfiguration(confProto);
+    } catch (FileNotFoundException e) {
+
+      return null;
     } catch (Exception e) {
-      LOG.error("Failed reading configuration from file:" + confFile, e);
       return null;
```

Comment:
I think we should keep the log for other exceptions, otherwise the two `catch` blocks can be merged.

Reply:
Or actually move the LOG.error to `FileNotFoundException` as that is a error that has happened thus printing that into log is useful?

I guess to avoid printing every exception is the goal of this PR?